Dependencies/UrlLib/Source/Unix/UrlRequest.cpp

```diff
@@ -60,16 +60,16 @@ namespace UrlLib
                     // So, escaping only path part of the URL for every URL but file scheme.
 
                     CURLUcode rc;
-                    CURLU* url = curl_url();
-                    auto urlScopeGuard = gsl::finally([url] { curl_url_cleanup(url); });
-                    rc = curl_url_set(url, CURLUPART_URL, url.c_str(), 0);
+                    CURLU* cURL_url = curl_url();
```

Comment:
nit: this is a weird name. it's mixed style (camel and snake case).
```suggestion
                    CURLU* urlObject = curl_url();
```

Reply:
Or
```suggestion
                    CURLU* curlUrl = curl_url();
```